pages/about.vue

```diff
@@ -1,102 +1,132 @@
+            </p>
+            <p
+                data-testid="about-paragraph1"
+                class="mb-6 text-md text-primary-text-muted"
+            >
+                {{ $t("about.paragraph1") }}
+            </p>
+            <p
+                data-testid="about-paragraph2"
+                class="text-md text-primary-text-muted"
+            >
+                {{ $t("about.paragraph2") }}
+            </p>
+        </div>
+        <div
+            data-testid="members-header-container"
+            id="members-header-container"
+            class="flex justify-center items-center"
+        >
+            <div
+                data-testid="members-header"
+                id="members-header"
+                class="flex w-4/5 my-14"
+            >
+                <div
+                    class="flex-1 border-currentColor/70 border self-center"
+                ></div>
+                <div
+                    class="text-primary/80 text-xl font-semibold mx-4 p-2 whitespace-nowrap"
+                >
+                    {{ $t("about.members") }}
+                </div>
+                <div
+                    class="flex-1 border-currentColor/70 border self-center"
+                ></div>
             </div>
-            <div class="members-list flex flex-row" :key="index" v-for="(member, index) in members">
-                <div class="member flex flex-col">
-                    <img :href="member.avatarImg" />
-                    <div class="member-name text-xl font-bold">{{ member.name }}</div>
-                    <div class="member-title text-md text-primary-text-muted">{{ member.title }}</div>
-                    <div class="member-social flex flex-row">
-                        <a :href="member.linkedInUrl" target="_blank" rel="noopener noreferrer">
-                            <svg role="img" alt="linkedin icon" title="linkedin icon"
-                                class="social-icon w-12 h-12 mr-1 fill-primary-text-muted">
-                                <use xlink:href="../assets/images/social-linkedin.svg#social-linkedin-svg" />
-                            </svg>
-                        </a>
-                        <a :href="member.githubUrl" target="_blank" rel="noopener noreferrer">
-                            <svg role="img" alt="github icon" title="github icon"
-                                class="social-icon w-12 h-12 mr-1 fill-primary-text-muted">
-                                <use xlink:href="../assets/images/social-github.svg#social-github-svg" />
-                            </svg>
-                        </a>
+        </div>
+        <div
+            data-testid="members-container"
+            id="members-container"
+            class="flex justify-center items-center mx-4"
+        >
+            <div
+                data-testid="members"
+                id="members"
+                class="flex flex-wrap justify-between items-start w-4/5 mx-4 md:flex-wrap mx-32"
```

Comment:
nit: we could use a little more whitespace at the bottom of the page after the members

```suggestion
                class="flex flex-wrap justify-between items-start w-4/5 mx-4 mb-8 md:flex-wrap mx-32"
```